build/proj/CoreBuild.proj

```diff
@@ -29,15 +29,31 @@
     <CommonBuildProperty Include="CIBuild=$(CIBuild)" />
     <CommonBuildProperty Include="RepoRoot=$(RepoRoot)" />
     <CommonBuildProperty Include="VersionsPropsPath=$(RepoRoot)build\import\Versions.props" />
+    <CommonBuildProperty Include="QuickSrcBuild=$(QuickSrcBuild)" />
   </ItemGroup>
 
   <ItemGroup>
     <SolutionRestoreAndBuildProperty Include="@(CommonBuildProperty)"/>
     <SolutionRestoreAndBuildProperty Include="__DeployProjectOutput=$(Deploy)" Condition="'$(Deploy)' != ''" />
   </ItemGroup>
 
-  <Target Name="LogInitialMessage">
-    <Message Text="Building $(SolutionFileName) [$(Configuration)]...%0a" Importance="high" />
+  <Target Name="GenerateSolutionForQuickSrcBuild" BeforeTargets="Restore" Condition="'$(QuickSrcBuild)' == 'true'">
+    <PropertyGroup>
+      <SolutionName>Src</SolutionName>
+      <SolutionFolder>$(RepoRoot)src/</SolutionFolder>
+      <SolutionFileName>$(SolutionName).sln</SolutionFileName>
+      <SolutionPath>$(SolutionFolder)$(SolutionFileName)</SolutionPath>
+    </PropertyGroup>
+    <Message Text="Generating Solution file [$(SolutionFileName)]..." Importance="high" />
+
+    <Exec Command="dotnet --version" />
+    <Exec Command="dotnet new sln -n $(SolutionName) -o $(SolutionFolder) --force" />
+
+    <ItemGroup>
+      <ProjFiles Include="$(SolutionFolder)**/*.*proj" />
+    </ItemGroup>
+    <!-- https://github.com/dotnet/cli/issues/6295#issuecomment-346973582 -->
+    <Exec Command="dotnet sln $(SolutionPath) add &quot;%(ProjFiles.FullPath)&quot;" />
```

Comment:
So here we're add all the project files (under src) to Src.sln, but where do we actually build Src.sln?

Reply:
The build is still done in the `Build` target. It runs `Src.sln` because we've changed the `SolutionFileName` and `SolutionPath` properties. You can see they default to `ProjectSystem.sln` at the top of the file. So, I'm just replacing them. No changes to the build itself were necessary.